database/migrations/2020_02_21_230554_add_experience_work_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddExperienceWorkTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experience_work', function (Blueprint $table) {
```

Comment:
Either make the table name plural (experience_works) so its what laravel expects, or specify the table name on the ExperienceWork model.